modules/administration-guide/partials/con_caching-images-for-faster-workspace-start.adoc

```diff
@@ -2,9 +2,7 @@
 [id="caching-images-for-faster-workspace-start_{context}"]
 = Caching images for faster workspace start
 
-To improve the start time performance of {prod-short} workspaces, use the {image-puller-name-short}. 
-The {image-puller-name-short} is an additional {platforms-name} deployment.
-It creates a _DaemonSet_ downloading and running the relevant container images on each node. These images are already available when a {prod-short} workspace starts.
+To improve the start time performance of {prod-short} workspaces, use the {image-puller-name-short}, a {prod-short}-agnostic component that can be used to pre-pull images for {platforms-name} clusters. The {image-puller-name-short} is an additional {platforms-name} deployment which creates a _DaemonSet_ that can be configured to pre-pull relevant {prod-short} workspace images on each node. These images would already be available when a {prod-short} workspace starts, therefore improving the workspace start time.
```

Comment:
What does "CRW-agnostic component" mean here?

Reply:
I've added that to specify that the image puller and Che/CRW aren't directly related in any way, since the image puller can be used in situations unrelated to Che/CRW. Maybe it's not necessary?